src/meta/src/stream/stream_manager.rs

```diff
@@ -59,6 +59,9 @@ pub struct CreateStreamingJobContext {
 
     /// The properties of the streaming job.
     pub table_properties: HashMap<String, String>,
+
+    /// DDL definition.
+    pub definition: String,
```

Comment:
I suggest we directly put the `stream_job` in the context, so that `definition` and `table_properties` can be covered. May refactor this in the future.